docs/spj_guide/src/asciidoc/_chapters/develop_spjs.adoc

```diff
@@ -83,19 +83,19 @@ PIC[TURE] X^1^ +
 NCHAR +
 NCHAR VARYING +
 NATIONAL CHAR[ACTER] +
-NATIONAL CHAR[ACTER] VARYING | java.lang.string
-| DATE | java.sql.date
-| TIME | java.sql.time
-| TIMESTAMP | java.sql.timestamp
+NATIONAL CHAR[ACTER] VARYING | java.lang.String
+| DATE | java.sql.Date
+| TIME | java.sql.Time
+| TIMESTAMP | java.sql.Timestamp
 | DEC[IMAL]^2^ +
 PIC[TURE] S9^3^ +
-NUMERIC (including numeric with a precision greater than eighteen)^2^ | java.math.bigdecimal
+NUMERIC (including numeric with a precision greater than eighteen)^2^ | java.math.Bigdecimal
```

Comment:
Oops... spotted a typo. It should be "java.math.BigDecimal" (note the capital D in the middle of things)

Reply:
@DaveBirdsall thanks, I've corrected 😁